requests_kerberos/kerberos_.py

```diff
@@ -158,6 +159,9 @@ def handle_401(self, response, **kwargs):
         log.debug("handle_401(): Handling: 401")
         if _negotiate_value(response) is not None:
             _r = self.authenticate_user(response, **kwargs)
+            if _r is None:
+                log.debug("handle_401(): can not find any kerberos ticket.")
+                raise RuntimeError("Can not find any kerberos ticket.")
```

Comment:
We shouldn't be raising a RuntimeError. We should be raising an exception from within the library.